templates/page.template

```diff
@@ -57,13 +130,53 @@
           </tr>
         </thead>
         <tbody>
-{{ servicelist(localservices) }}
+          {{ servicelist(localservices) }}
         </tbody>
       </table>
     </div>
-    <footer class="footer mt-auto py-3">
-     <img src="{{ logo }}" style="max-width: 30%; height: auto;"><span style="float: right;"><a href="https://github.com/samply/bridgehead"><button type="button" class="btn btn-primary">Documentation</button></a></span>
-    </footer>
+  </div>
+
+  <footer class="footer mt-auto py-3">
+    <img src="{{ logo }}" alt="Logo">
+    <div>
+      <a href="https://github.com/samply/bridgehead">
+        <button type="button" class="btn btn-primary">Documentation</button>
+      </a>
+      <!-- Button trigger modal -->
+<button type="button" class="btn btn-primary" data-bs-toggle="modal" data-bs-target="#contactModal">
+  Contact
+</button>
+
+<!-- Modal -->
```

Comment:
Please make the modal larger and add some text like "Need help, contact us under <MAIL>"